src/handlers/assign.rs

```diff
@@ -448,10 +451,12 @@ pub(super) async fn handle_command(
             AssignCommand::Claim => event.user().login.clone(),
             AssignCommand::AssignUser { username } => username,
             AssignCommand::ReleaseAssignment => {
-                log::trace!(
-                    "ignoring release on PR {:?}, must always have assignee",
-                    issue.global_id()
-                );
+                let current = &event.user().login;
+                if issue.contain_assignee(current) {
+                    issue
+                        .remove_assignees(&ctx.github, Selection::One(&current))
+                        .await?;
+                }
```

Comment:
Don't we want to maintain our rule of at least one assignee per PR? If I'm not mistaken this will allow the current assignee to remove them-selves but without selecting anyone else.

How about restarting the selection process with the current assignee excluded?

Reply:
While there are triagebot comments that suggest so, it's not really a rule today. If you open a PR with r? @ghost, it won't be assigned to anyone, so that is existing practice. This change mostly serves to support the use-case of "oh shit, I forgot to add r? @ghost to the PR body, let me fix this".

Reply:
It might be only me but I see `r? ghost` and `release-assignment` as different, the first being "don't assign this PR to anyone" for which doing `r? ghost` after the fact achieve that, while `release-assignment` "Removes your assignment" (per [documentation](https://forge.rust-lang.org/triagebot/index.html#common-command-summary)) doesn't *to me* means remove all assignments.

Reply:
Right, and this is how it behaves in this PR, right? release-assignment only releases the comment author. I guess that you propose to "reroll" an assignee after you do release-assignment? That sounds reasonable, but I would rather do it in a separate PR. I will remove the release-assignment changes from this PR, and keep only the ghost change. Is that fine with you?

Reply:
:+1: to postponing release-assignment.